server/routes/courseRoutes.mjs

```diff
@@ -142,7 +143,7 @@ router.get('/:uid', verifyToken, getCourse);
  *       500:
  *         description: Internal server error.
  */
-router.post('/', verifyToken, isTeacher, createCourse);
+router.post('/', verifyToken, isTeacher, multer({ dest: './public/data/uploads/' }).single('logo'), createCourse);
 
```

Comment:
Nit: While there’s nothing wrong with having the `multer` configuration here, it would be cleaner to move it to a separate middleware file. This would keep our routes tidy and make it easier to expand the logic, like adding the cleanup process I mentioned earlier in courseController.mjs, setting file size limits to control storage use, etc.